src/components/two-factor-authentication/HardwareTokenWorkflow.vue

```diff
@@ -0,0 +1,149 @@
+<script setup lang="ts">
+  import { useDisplay } from 'vuetify';
+  import FormRow from '@/components/form/FormRow.vue';
+  import { ref, type Ref } from 'vue';
+  import { usePersonStore, type Personendatensatz, type PersonStore } from '@/stores/PersonStore';
+  import axios from 'axios';
+  import { useI18n, type Composer } from 'vue-i18n';
+
+  const { t }: Composer = useI18n({ useScope: 'global' });
+  const { mdAndDown }: { mdAndDown: Ref<boolean> } = useDisplay();
+  const dialogText: Ref<string> = ref('');
+  const proceeded: Ref<boolean> = ref(false);
+  const errorThrown: Ref<boolean> = ref(false);
+  const personStore: PersonStore = usePersonStore();
+  // eslint-disable-next-line @typescript-eslint/typedef
+  const emits = defineEmits(['updateHeader', 'onCloseClicked']);
+  emits('updateHeader', 'Hardware-Token zuordnen');
```

Comment:
Bitte einen Typ Emits definieren, anstatt ESLint zu disablen. Beispiel in ResultTable.vue

Reply:
done